### PR TITLE
Updated old FAQ questions

### PR DESCRIPTION
**Pull request by SolusBellator**
Updated the old FAQ questions, deleted those not up to date, and added a link to the code of conduct to the FAQ page.

### CODE REVIEW

File: src/components/Faq.astro [hunk 1]
```diff
@@ -15,7 +15,7 @@ import faq from '../assets/data/faq.json';
           Can't find the answer you're looking for? You can always <a
             href="mailto:team@hackhpi.org"
             class="font-medium text-hackhpi-purple">send us an email</a
-          > with your enquiry.
+          > with your enquiry. Furthermore, you can find the MLH's code of conduct <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">here</a>.
```
Comment (Benjamin-Frost) [inline src/components/Faq.astro:18]:
```suggestion
          > with your enquiry. Furthermore, you can find the MLH's code of conduct
          <a
            href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf"
            target="_blank"
            rel="noopener noreferrer"
            class="font-medium text-hackhpi-purple">here</a
          >.
```

**Review by Benjamin-Frost — CHANGES_REQUESTED** [review]
Really well done 🚀  Just a few suggestions

File: src/assets/data/faq.json [hunk 2]
```diff
@@ -5,59 +5,55 @@
...
   },
   {
-    "question": "How is the application process?",
-    "answer": "You can apply over the form below, please only do so if you are planning to attend the hackathon, as you might otherwise take the place of somebody else. We will then evaluate your application and inform everyone at the end of February. Even if you are not selected at the beginning you might have the chance to move up after someone else jumps up. Should you need a faster response, indicate it in the form and we will try to contact you faster."
+    "question": "How does the application process work?",
+    "answer": "You can apply by filling all required information into the form below. We kindly ask you to only do so if you are planning to attend the hackathon, as you might otherwise take someone else's place. We will then evaluate your application and inform everyone by the end of February. Even if you are not selected at the beginning, you might have the chance to move up in our listings if someone else decides to leave. In cases where you need a prioritized response, indicate it in the form and we will try to contact you faster."
```
[the remark this answers is not in the document]
Comment (SolusBellator) [reply src/assets/data/faq.json:24]: Didn't change this one for now, expected that the page structure would remain the same and the mentioned form would be added where it has obviously been when this FAQ was reworked last time.

File: src/assets/data/faq.json [hunk 3]
```diff
@@ -5,59 +5,55 @@
...
   {
     "question": "Is there a code of conduct?",
-    "answer": "Yes. We follow MLH's code of conduct, which all attendees, partners, volunteers, and staff are required to agree to. We expect all of our attendees to be respectful and considerate of others."
+    "answer": "Yes. We comply to the MLH's code of conduct, which all attendees, partners, volunteers, and staff are required to agree to. We expect all of our attendees to be respectful and considerate of others. For your information, it is linked above this FAQ."
   },
   {
-    "question": "What are the rules concerning Corona?",
-    "answer": "We plan our hackathon as a 2g+ event. Meaning that every participant needs to be vaccinated or recovered. We will provide rapid self-tests, which every participant is required to do before the event."
+    "question": "What are the rules concerning COVID?",
+    "answer": "as of right now, there are only few rules concerning COVID regulations. We strongly recommend you to wear a face mask and to test yourself before the event, yet we can also provide you with a mask if you don't have any. No vaccination or recovery certificates required."
```
[the remark this answers is not in the document]
Comment (SolusBellator) [reply src/assets/data/faq.json:20]: dict.cc gives me the following 2 results that fit:

- to recommend doing sth.
  Means gerund would be used: "We recommend wearing a face mask"

- to recommend **sb.** _to_ do sth
  Means _to_ + _**infinitive**_: "We recommend **you** _to_ **_wear_** a face mask"

I've decided for the second option for I personally prefer it, yet oth are correct. What is unfortunately wrong or at least absolutely bad style (sorry to say), is your recommendation here.

File: src/assets/data/faq.json [hunk 4]
```diff
@@ -5,59 +5,55 @@
...
   },
   {
     "question": "Will the Hackathon be in present?",
-    "answer": "We believe that Hackathons need the human contact that only in-person events can provide. We are planning firmly that we will be able to conduct the Hackathon with our hygiene concept in presence."
+    "answer": "Yes, and that's quite a sure thing. We believe that Hackathons need the human contact that only in-person events can provide, and we're glad to announce that the HackHPI 2023 will be an in-presence event!"
   },
   {
     "question": "Is there a code of conduct?",
-    "answer": "Yes. We follow MLH's code of conduct, which all attendees, partners, volunteers, and staff are required to agree to. We expect all of our attendees to be respectful and considerate of others."
+    "answer": "Yes. We comply to the MLH's code of conduct, which all attendees, partners, volunteers, and staff are required to agree to. We expect all of our attendees to be respectful and considerate of others. For your information, it is linked above this FAQ."
```
[the remark this answers is not in the document]
Comment (SolusBellator) [reply src/assets/data/faq.json:16]: Yup, that's an actual mistake. I'll fix that.

File: src/assets/data/faq.json [hunk 5]
```diff
@@ -5,59 +5,55 @@
...
   },
   {
     "question": "Are there any requirements?",
-    "answer": "We believe that diverse teams come up with more creative solutions. A hackathon should be a way to experience teamwork and work on your abilities. We, therefore, encourage every major to apply and welcome also young professionals. However, due to legal reasons, every participant has to be 18 years old or older."
+    "answer": "We believe that diverse teams come up with more creative solutions. A hackathon should be a way to experience teamwork and work on your abilities. We, therefore, encourage every major to apply, and give a warm welcome to young professionals as well. However, due to german law, every participant has to be at least 18 years old."
```
[the remark this answers is not in the document]
Comment (SolusBellator) [reply src/assets/data/faq.json:28]: Actually there's a different comma mistake here I just noticed whilst reviewing the review: "therefore" should not be encapsulated with commas, this breaks up sentence structure and makes it an independent sentence, which results in multiple grammatical issues. The comma before the and is an intonation indicator that is not mandatory but makes things easier to read and/or speak.

File: src/assets/data/faq.json [hunk 6]
```diff
@@ -5,59 +5,55 @@
...
-    "answer": "Nope! HackHPI is completely free, we provide you with food, drinks and a lot of caffeine during the whole Hackathon."
+    "answer": "Nope! HackHPI is completely free, and we provide you with food, drinks (and a lot of caffeine! :D) during the whole Hackathon."
   },
   {
     "question": "Where can I stay during the Hackathon?",
-    "answer": "This is an all-night Hackathon. Meaning that it goes all night long. We will provide rooms in which you can sleep, but you need to bring your own sleeping bag and air mattress. We are also planning on organizing a couch surfing option from Friday to Saturday for everybody coming early, but can not guarantee that this option is available for everyone."
+    "answer": "This is an all-night Hackathon, meaning that we won't stop until the last one standig. We will provide rooms in which you can sleep or just rest, but you're required to bring your own sleeping bag and air mattress. We're also planning on organizing a couch surfing option from Friday to Saturday for everybody coming early. If you wish to participate in this, wether you want to stay or offer a bed, please note that down when filling in the application form."
```
[the remark this answers is not in the document]
Comment (SolusBellator) [reply src/assets/data/faq.json:44]: Whether is a typo.

I've used gerund here to create an allusion to the famous "game mode design" of "Last man standing", and as it isn't wrong, it oukd stay there.

Unfortunately, your suggestion isn't wrong, but gives the whole sentence a completely different (and actually inverted) meaning: "not stop until the last one stands" means that everyone is NOT standing at the beginning, and gets up over time. What I was referring to here is, as stated above, the "last man standing" principle, where you try to keep up for as long as possible (which kinda fits the idea of an overnight hackathon)

File: src/components/Faq.astro (hunk 1 above)
Comment (SolusBellator) [reply src/components/Faq.astro:18]: Sure, I haven't used Astro before, and kept it at basic HTML, so this is your part, I'm not qualified to evaluate this.

**Pull request closed without merging**